src/octoprint/static/js/app/viewmodels/control.js

```diff
@@ -524,6 +526,8 @@ $(function () {
                 self._switchToMjpgWebcam();
             } else if (streamType == "hls") {
                 self._switchToHlsWebcam();
+            } else if (streamType == "webrtc") {
```

Comment:
This needs a compatibility check to determine if webrtc is supported. Only then should we attempt to switch.